inlong-dashboard/package-lock.json

```diff
@@ -15797,7 +15797,7 @@
       "dev": true
     },
     "vite": {
-      "version": "3.0.9",
+      "version": "3.2.7",
```

Comment:
It is incomplete to modify `package-lock.json` directly, and `vite` is also a dependent package in `package.json`.

You may need to have more modifications.